tests/tests_luks.yml

```diff
@@ -15,6 +15,20 @@
         min_size: "{{ volume_size }}"
         max_return: 1
 
+    - name: Create a key file
+      tempfile:
+        prefix: storage_test
+        suffix: lukskey
+      register: storage_test_key_file
+      changed_when: false
+
+    - name: Write the key into the key file
+      lineinfile:
```

Comment:
I would suggest using `copy` instead of `lineinfile` (the current best practice is to avoid `lineinfile` if possible - https://github.com/oasis-roles/meta_standards#ansible-best-practices)
```yaml
     - name: Write the key into the key file
       copy:
         dest: "{{ storage_test_key_file.path }}"
         content: veryveryveryveryverysecureindeed
       changed_when: false
```

Reply:
Fixed in latest version in both `tests_luks.yml` and `tests_luks_pool.yml`.